QModManager/API/ModLoading/Internal/Initializer.cs

```diff
@@ -0,0 +1,127 @@
+﻿namespace QModManager.API.ModLoading
```

Comment:
The `Internal` folder inside of `QModManager/API/ModLoading` should be moved someplace else.